Docs/articles/Kernel/CGS.md

```diff
@@ -41,10 +41,13 @@ namespace GraphicTest
         protected override void BeforeRun()
         {
             /* If all works correctly you should not really see this :-) */
-            Console.WriteLine("Cosmos booted successfully. Let's go in Graphic Mode");
+            Console.WriteLine("Cosmos booted successfully. Let's go in Graphical Mode");
 
             canvas = FullScreenCanvas.GetFullScreenCanvas();
 
+            /* It is better to directly specify the mode in the GetFullScreenCanvas() function */
+            canvas.Mode = new Mode(640, 480, ColorDepth.ColorDepth32);
```

Comment:
Can you please pass the mode to GetFullScreenCanvas directly? Setting the Mode has been causing problems in VMWare iirc.

Reply:
This was only to demonstrate, but sure.